blockchain/vm/evm.go

```diff
@@ -538,6 +538,33 @@ func (evm *EVM) CreateWithAddress(caller types.ContractRef, code []byte, gas uin
 	return evm.create(caller, codeAndHash, gas, value, contractAddr, humanReadable, codeFormat)
 }
 
+func (evm *EVM) CurrentCodeFormat() params.CodeFormat {
+	switch {
+	case evm.chainRules.IsIstanbul:
+		return params.CodeFormatEVM2
+	default:
+		return params.CodeFormatEVM
+	}
+}
+
+func (evm *EVM) GetPrecompiledContract(caller, address common.Address) map[common.Address]PrecompiledContract {
```

Comment:
I think `GetPrecompiledContractMap` is more proper for a name.

Reply:
Thanks, it's a good suggestion. Reflected at https://github.com/klaytn/klaytn/pull/912/commits/1c0928ba549158af0d357eed0e6d0a7763e14bd8.